cfgov/jinja2/v1/_includes/organisms/carousel.html

```diff
@@ -35,9 +31,11 @@ <h2>{{ item.title }}</h2>
         <p><a href="{{ item.link.url }}">{{ item.link.text }}</a></p>
     </div>
     <div class="o-carousel_item-visual">
+        {% set image_atom = make_image_atom( item.image ) %}
+        {% set original_image = image( image_atom.upload, 'original' ) %}
         <img class="o-carousel_item-img"
-             src="{{ item.image.url }}"
-             alt="{{ item.image.alt_text }}"
+             src="{{ original_image.url }}"
```

Comment:
TRIVIAL: Maybe we should follow general -> specific naming and have `image_original`, `image_atom`, `image_thumbnail`, etc.